### PR TITLE
Updating the ubuntu vulnsrc to use the new git repo

### PR DESCRIPTION
**Pull request by MackJM**
Hi,
Since ubuntu has officially moved their cve-tracker to use git instead of bzr. I've updated the vulnsrc for ubuntu to use git. A lot of the code was copied from Alpine.

I've built and tested locally and it all runs fine.

Cheers,

### CODE REVIEW

File: ext/vulnsrc/ubuntu/ubuntu.go [hunk 1]
```diff
@@ -162,120 +160,81 @@ func (u *updater) Clean() {
...
+	if err != nil {
+		return "", vulnsrc.ErrGitFailure
 	}
 
-	return nil
+	commit = strings.TrimSpace(string(out))
+	return
 }
 
 func getRevisionNumber(pathToRepo string) (int, error) {
```
Comment (jzelinskie) [inline ext/vulnsrc/ubuntu/ubuntu.go]: Why didn't you delete this whole function?

**Review by jzelinskie — CHANGES_REQUESTED** [review]
Everything else looks good!

File: ext/vulnsrc/ubuntu/ubuntu.go (hunk 1 above)
Comment (MackJM) [reply ext/vulnsrc/ubuntu/ubuntu.go]: I missed it :disappointed:... I'll remove it

File: ext/vulnsrc/ubuntu/ubuntu.go [hunk 2]
```diff
@@ -162,120 +160,76 @@ func (u *updater) Clean() {
...
+		return "", vulnsrc.ErrGitFailure
 	}
 
-	return revno, nil
+	commit = strings.TrimSpace(string(out))
+	return
 }
 
-func collectModifiedVulnerabilities(revision int, dbRevision, repositoryLocalPath string) (map[string]struct{}, error) {
+func collectModifiedVulnerabilities(revision, dbRevision, repositoryLocalPath string) (map[string]struct{}, error) {
```
Comment (jzelinskie) [inline ext/vulnsrc/ubuntu/ubuntu.go]: Rename revision "commit" and get rid of dbRevision since we aren't using it anymore.

**Review by jzelinskie — CHANGES_REQUESTED** [review]

**Review by jzelinskie — APPROVED** [review]

**Comment by jzelinskie** [comment]
Just got done testing this locally -- thanks sooo much for this PR! I also did the work for the master branch. Bzr has caused so many issues in the past, I'm glad we can finally get rid of it.

**Comment by MackJM** [comment]
Glad I could help. I wasn't sure if I should do this on master or release-2.0... Happy I picked right :smile:

**Pull request merged**